index.js

```diff
@@ -90,12 +97,26 @@ client.on('message', async (channel, tags, message, self) => {
             } else {
                 await handleSongRequest(channel, tags[displayNameTag], message, true);
             }
-    } else if (messageToLower === chatbotConfig.skip_alias) {
+    } else if (chatbotConfig.allow_volume_set && messageToLower.split(" ")[0] == '!volume') {
+        let args = messageToLower.split(" ")[1];
+            if (!args) {
+                await handleGetVolume(channel, tags);
+            } else {
+                await handleSetVolume(channel, tags, args);
+            }
+    } 
+    else if (messageToLower === chatbotConfig.skip_alias) {
         await handleSkipSong(channel, tags);
     }
     else if (chatbotConfig.use_song_command && messageToLower === '!song') {
         await handleTrackName(channel);
     }
+	else if (chatbotConfig.use_queue_command && messageToLower === '!queue') {
```

Comment:
Same comment as line 100

Reply:
!song is a previously established command and therefore we are following previously established coding standard.
![image](https://user-images.githubusercontent.com/22343767/201497004-2f254c8d-5115-4e3f-a32b-5e8f802d6c93.png)